pkg/build/builder/cmd/builder.go

```diff
@@ -50,6 +50,7 @@ func run(builderFactory factoryFunc) {
 				glog.Fatalf("Cannot setup secret file for accessing private repository: %v", err)
 			}
 			if sourceURL != nil {
+				build.Annotations[bld.OriginalSourceURLAnnotationKey] = build.Spec.Source.Git.URI
```

Comment:
yeah, what was the purpose of this... it seems like we were trying to squirrel away the auth info, but now we're resetting it as an annotation? seems weird